client/app/house/components/DynamicRuleField.tsx

```diff
@@ -0,0 +1,91 @@
+import { useFieldArray, useFormContext, useWatch } from 'react-hook-form'
+import { FiX } from 'react-icons/fi'
+import { FiPlus } from 'react-icons/fi'
+import {
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  Input,
+  Button,
+} from '@/_components/ui'
+import { CreateHouseSchema } from '../schema'
+
+const DynamicRuleField = () => {
+  const { control } = useFormContext<CreateHouseSchema>()
+
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: 'rules',
+  })
+
+  const value = useWatch({
+    name: 'rules',
+    control,
+  })
+
+  return (
+    <div className='flex flex-col gap-4'>
+      <FormField
+        control={control}
+        name='rules'
+        render={() => (
+          <FormItem>
+            <FormLabel>Rules</FormLabel>
+
+            <div className='flex flex-col gap-2'>
+              {fields.map((_, index) => (
+                <div key={index} className='flex gap-4'>
+                  <FormField
+                    control={control}
+                    name={`rules.${index}.text`}
+                    render={({ field }) => (
+                      <FormItem className='w-full'>
+                        <FormControl>
+                          <Input
+                            placeholder='Clean the kitchen after you cook'
+                            {...field}
+                            value={value[index]?.text || ''}
+                            className=''
```

Comment:
Empty className!

Reply:
Deleted it!
https://github.com/HAK2024/fairy-share/pull/81/commits/739296a060edd606f24d5b8c1e068dfde595d94b